src/dropbox.ml

```diff
@@ -190,6 +204,16 @@ module type S = sig
                  ?include_deleted: bool -> ?rev: string -> ?locale: string ->
                  ?include_media_info: bool -> ?include_membership: bool ->
                  string -> metadata option Lwt.t
+
+  val delta : ?cursor: string -> ?locale: string -> ?path_prefix: string
+              -> ?include_media_info: bool -> t -> delta Lwt.t
+
+  val latest_cursor : ?path_prefix: string -> ?include_media_info: bool
+                      -> t -> delta Lwt.t
```

Comment:
On Sun, 26 Apr 2015 15:34:05 -0700, elaquentil wrote:
> 
> In src/dropbox.ml:
> 
> > @@ -190,6 +204,16 @@ module type S = sig
> >                   ?include_deleted: bool -> ?rev: string -> ?locale: string ->
> >                   ?include_media_info: bool -> ?include_membership: bool ->
> >                   string -> metadata option Lwt.t
> > +
> > +  val delta : ?cursor: string -> ?locale: string -> ?path_prefix: string
> > +              -> ?include_media_info: bool -> t -> delta Lwt.t
> > +
> > +  val latest_cursor : ?path_prefix: string -> ?include_media_info: bool
> > +                      -> t -> delta Lwt.t
> 
> Indeed, I thought it would be a gain of place to take the same dictionary as
> delta function.

Wait for my next push.  I think this will be conceptually cleaner.